accounting/src/creditcommons/credit-commons-controller.ts

```diff
@@ -35,9 +55,72 @@ export class CreditCommonsControllerImpl extends AbstractCurrencyController impl
     if (record.lastHash !== ctx.lastHashAuth?.lastHash) {
       throw unauthorized(`value of last-hash header ${JSON.stringify(ctx.lastHashAuth?.lastHash)} does not match our records.`)
     }
+    return record.vostroId
   }
   async getWelcome(ctx: Context) {
     await this.checkLastHashAuth(ctx)
     return { message: 'Welcome to the Credit Commons federation protocol.' }
   }
+  async getAccountHistory(ctx: Context) {
+    await this.checkLastHashAuth(ctx)
+    return {
+      data: {
+        '2025-04-01 09:22:37': 0,
```

Comment:
If I'm getting it correctly, this is just fake data. However, shouldn't we use ISO 8601 for times? Like `2025-04-01T09:22:37Z`, for example. Is credit commons using another notation?

Reply:
This comes from https://gitlab.com/credit-commons/cc-php-lib/-/blob/0.9.x/docs/credit-commons-openapi3.yml?ref_type=heads&plain=1#L859-867
I created https://gitlab.com/credit-commons/cc-php-lib/-/issues/2 about it